### ISSUE TITLE
[FORGE] [SERVER] [MOHIST] Server won't load with pehkui forge mod

### ISSUE THREAD

**Issue by Matthew-MBG** [Virtuoel/Pehkui#84]
Error:

```
Mohist is running with custom Java (for Windows) | Launch Command - [C:\Users\matth\Desktop\Mohist Server\CustomJAVA/bin/java.exe, -jar, server.jar, launchedWithCustomJava8]


 __    __   ______   __  __   __   ______   ______
/\ "-./  \ /\  __ \ /\ \_\ \ /\ \ /\  ___\ /\__  _\
\ \ \-./\ \\ \ \/\ \\ \  __ \\ \ \\ \___  \\/_/\ \/
 \ \_\ \ \_\\ \_____\\ \_\ \_\\ \_\\/\_____\  \ \_\
  \/_/  \/_/ \/_____/ \/_/\/_/ \/_/ \/_____/   \/_/


                                      Welcome to Mohist 1.16.5 ! - Mohist Developement Group
Checking libraries, please wait...
All libraries are nice :) !
Checking the installation, please wait.
Finished the install verification process !
Update in check...
If you don't want to enable update check, set check_update in mohist-config/mohist.yml to false
The latest Mohist version is 1.16.5-505 but you have 1.16.5-492. The latest version has been built on 2021-04-21 15:06:59. You should update to fix recent bugs !
[14:22:11 INFO]: ModLauncher running: args [--gameDir, ., --launchTarget, fmlserver, --fml.forgeVersion, 36.1.4, --fml.mcpVersion, 20210115.111550, --fml.mcVersion, 1.16.5, --fml.forgeGroup, net.minecraftforge, launchedWithCustomJava8]
[14:22:11 INFO]: ModLauncher 8.0.9+86+master.3cf110c starting: java version 11.0.10 by AdoptOpenJDK
[14:22:12 INFO]: Added Lets Encrypt root certificates as additional trust
[14:22:12 INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/C:/Users/matth/Desktop/Mohist%20Server/libraries/org/spongepowered/mixin/0.8.2/mixin-0.8.2.jar Service=ModLauncher Env=SERVER
[14:22:13 INFO]: [jdk.nashorn.api.scripting.NashornScriptEngine:<init>:-1]: Warning: Nashorn engine is planned to be removed from a future JDK release
[14:22:13 INFO]: [jdk.nashorn.api.scripting.NashornScriptEngine:<init>:-1]: Warning: Nashorn engine is planned to be removed from a future JDK release
[14:22:13 INFO]: [jdk.nashorn.api.scripting.NashornScriptEngine:<init>:-1]: Warning: Nashorn engine is planned to be removed from a future JDK release
[14:22:16 INFO]: Launching target 'fmlserver' with arguments [--gameDir, ., launchedWithCustomJava8]
[14:22:19 ERROR]: Critical injection failure: LVT in net/minecraft/entity/Entity::func_70099_a(Lnet/minecraft/item/ItemStack;F)Lnet/minecraft/entity/item/ItemEntity; has incompatible changes at opcode 116 in callback net/minecraft/entity/Entity::onDropStack.
Expected: [Lnet/minecraft/entity/item/ItemEntity;]
   Found: [Z]
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:39)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.Launcher.run(Launcher.java:82)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.Launcher.main(Launcher.java:66)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at net.minecraftforge.server.ServerMain$Runner.runLauncher(ServerMain.java:113)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at net.minecraftforge.server.ServerMain$Runner.access$100(ServerMain.java:110)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at net.minecraftforge.server.ServerMain.main(ServerMain.java:107)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]: Caused by: java.lang.reflect.InvocationTargetException
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at java.base/java.lang.reflect.Method.invoke(Unknown Source)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at net.minecraftforge.fml.loading.FMLServerLaunchProvider.lambda$launchService$0(FMLServerLaunchProvider.java:51)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37)
[14:22:19 INFO]: [java.lang.ThreadGroup:uncaughtException:-1]:  ... 7 more
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]: Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:223)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinTransformationHandler.processClass(MixinTransformationHandler.java:121)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.launch.MixinLaunchPlugin.processClass(MixinLaunchPlugin.java:131)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:154)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:85)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:265)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:136)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:98)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at net.minecraft.server.MinecraftServer.<clinit>(MinecraftServer.java:180)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at net.minecraft.server.Main.main(Main.java:104)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      ... 13 more
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]: Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: LVT in net/minecraft/entity/Entity::func_70099_a(Lnet/minecraft/item/ItemStack;F)Lnet/minecraft/entity/item/ItemEntity; has incompatible changes at opcode 116 in callback net/minecraft/entity/Entity::onDropStack.
Expected: [Lnet/minecraft/entity/item/ItemEntity;]
   Found: [Z]
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:487)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:437)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:264)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:385)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1284)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1042)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:395)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:320)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:345)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:569)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:351)
[14:22:19 INFO]: [java.lang.Throwable:printStackTrace:-1]:      ... 25 more
```

**Comment by QPCrummer** [comment]
I had this problem too, I posted an issue on Mohist's Github.

**Comment by Kraftygames** [comment]
Mod File: Pehkui-2.0.0+1.16.5-forge.jar

The mod works well on singleplayer, but whenever I integrate it with a Forge server: it always outputs "Connection closed - mismatched mod channel list" consistent on 6 different trials in conjunction with other mods and with the mod alone.

> [22Apr2021 18:33:30.903] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--gameDir, ., --launchTarget, fmlserver, --fml.forgeVersion, 36.1.0, --fml.mcpVersion, 20210115.111550, --fml.mcVersion, 1.16.5, --fml.forgeGroup, net.minecraftforge, nogui]
[22Apr2021 18:33:30.905] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 8.0.9+86+master.3cf110c starting: java version 1.8.0_271 by Oracle Corporation
[22Apr2021 18:33:31.213] [main/INFO] [net.minecraftforge.fml.loading.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[22Apr2021 18:33:31.242] [main/INFO] [mixin/]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/D:/Minecraft/server/pehkui%20test/libraries/org/spongepowered/mixin/0.8.2/mixin-0.8.2.jar Service=ModLauncher Env=SERVER
[22Apr2021 18:33:31.273] [main/WARN] [net.minecraftforge.fml.loading.FMLConfig/CORE]: Configuration file D:\Minecraft\server\pehkui test\config\fml.toml is not correct. Correcting
[22Apr2021 18:33:31.278] [main/WARN] [net.minecraftforge.fml.loading.FMLConfig/CORE]: Incorrect key [defaultConfigPath] was corrected from null to defaultconfigs
[22Apr2021 18:33:31.969] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'fmlserver' with arguments [--gameDir, ., nogui]
[22Apr2021 18:33:36.444] [modloading-worker-11/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 36.1.0, for MC 1.16.5 with MCP 20210115.111550
[22Apr2021 18:33:36.449] [modloading-worker-11/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v36.1.0 Initialized
[22Apr2021 18:33:36.692] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file D:\Minecraft\server\pehkui test\config\forge-common.toml is not correct. Correcting
[22Apr2021 18:33:36.693] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general was corrected from null to its default, SimpleCommentedConfig:{}. 
[22Apr2021 18:33:36.694] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.defaultWorldType was corrected from null to its default, default. 
[22Apr2021 18:33:36.707] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file D:\Minecraft\server\pehkui test\config\pehkui-common.toml is not correct. Correcting
[22Apr2021 18:33:36.709] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general was corrected from null to its default, SimpleCommentedConfig:{}. 
[22Apr2021 18:33:36.710] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledFallDamage was corrected from null to its default, true. 
[22Apr2021 18:33:36.710] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledMotion was corrected from null to its default, true. 
[22Apr2021 18:33:36.724] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledReach was corrected from null to its default, true. 
[22Apr2021 18:33:36.725] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledAttack was corrected from null to its default, true. 
[22Apr2021 18:33:36.725] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledDefense was corrected from null to its default, true. 
[22Apr2021 18:33:36.726] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledHealth was corrected from null to its default, true. 
[22Apr2021 18:33:36.726] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledItemDrops was corrected from null to its default, true. 
[22Apr2021 18:33:36.727] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledProjectiles was corrected from null to its default, true. 
[22Apr2021 18:33:36.727] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scaledExplosions was corrected from null to its default, true. 
[22Apr2021 18:33:36.727] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.keepAllScalesOnRespawn was corrected from null to its default, false. 
[22Apr2021 18:33:36.728] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.accurateNetherPortals was corrected from null to its default, true. 
[22Apr2021 18:33:36.728] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.scalesKeptOnRespawn was corrected from null to its default, []. 
[22Apr2021 18:33:36.729] [main/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key general.largeScaleCollisionThreshold was corrected from null to its default, 26.0. 
[22Apr2021 18:33:36.759] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Starting version check at https://files.minecraftforge.net/maven/net/minecraftforge/forge/promotions_slim.json
[22Apr2021 18:33:38.226] [main/INFO] [com.mojang.authlib.yggdrasil.YggdrasilAuthenticationService/]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[22Apr2021 18:33:38.661] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [scale, delay, get, scale_type] and [scale, delay, get, entity] with inputs: [base]
[22Apr2021 18:33:38.664] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [scale, get, scale_type] and [scale, get, entity] with inputs: [base]
[22Apr2021 18:33:38.682] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [scale, reset, scale_type] and [scale, reset, targets] with inputs: [base]
[22Apr2021 18:33:38.689] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, destination] and [teleport, targets] with inputs: [Player, 0123, @e, dd12be42-52a9-4a91-a8a1-11c01849e498]
[22Apr2021 18:33:38.693] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[22Apr2021 18:33:38.696] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, location] and [teleport, targets] with inputs: [0.1 -0.5 .9, 0 0 0]
[22Apr2021 18:33:38.698] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets] and [teleport, destination] with inputs: [Player, 0123, dd12be42-52a9-4a91-a8a1-11c01849e498]
[22Apr2021 18:33:38.701] [main/WARN] [net.minecraft.command.Commands/]: Ambiguity between arguments [teleport, targets, location] and [teleport, targets, destination] with inputs: [0.1 -0.5 .9, 0 0 0]
[22Apr2021 18:33:38.711] [main/INFO] [net.minecraft.resources.SimpleReloadableResourceManager/]: Reloading ResourceManager: Default, Pehkui-2.0.0+1.16.5-forge.jar, forge-1.16.5-36.1.0-universal.jar
[22Apr2021 18:33:39.139] [Worker-Main-11/INFO] [net.minecraft.item.crafting.RecipeManager/]: Loaded 7 recipes
[22Apr2021 18:33:39.275] [Forge Version Check/INFO] [net.minecraftforge.fml.VersionChecker/]: [forge] Found status: UP_TO_DATE Current: 36.1.0 Target: null
[22Apr2021 18:33:39.645] [Worker-Main-11/INFO] [net.minecraft.advancements.AdvancementList/]: Loaded 927 advancements
[22Apr2021 18:33:40.302] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting minecraft server version 1.16.5
[22Apr2021 18:33:40.303] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Loading properties
[22Apr2021 18:33:40.306] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Default game type: SURVIVAL
[22Apr2021 18:33:40.306] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Generating keypair
[22Apr2021 18:33:40.390] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Starting Minecraft server on *:25566
[22Apr2021 18:33:40.396] [Server thread/INFO] [net.minecraft.network.NetworkSystem/]: Using default channel type
[22Apr2021 18:33:40.639] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[22Apr2021 18:33:40.639] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: The server will make no attempt to authenticate usernames. Beware.
[22Apr2021 18:33:40.641] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[22Apr2021 18:33:40.641] [Server thread/WARN] [net.minecraft.server.dedicated.DedicatedServer/]: To change this, set "online-mode" to "true" in the server.properties file.
[22Apr2021 18:33:40.664] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file .\world\serverconfig\forge-server.toml is not correct. Correcting
[22Apr2021 18:33:40.665] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server was corrected from null to its default, SimpleCommentedConfig:{}. 
[22Apr2021 18:33:40.666] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.removeErroringEntities was corrected from null to its default, false. 
[22Apr2021 18:33:40.666] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.removeErroringTileEntities was corrected from null to its default, false. 
[22Apr2021 18:33:40.667] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.fullBoundingBoxLadders was corrected from null to its default, false. 
[22Apr2021 18:33:40.667] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.zombieBaseSummonChance was corrected from null to its default, 0.1. 
[22Apr2021 18:33:40.668] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.zombieBabyChance was corrected from null to its default, 0.05. 
[22Apr2021 18:33:40.668] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.logCascadingWorldGeneration was corrected from null to its default, true. 
[22Apr2021 18:33:40.668] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.fixVanillaCascading was corrected from null to its default, false. 
[22Apr2021 18:33:40.669] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.dimensionUnloadQueueDelay was corrected from null to its default, 0. 
[22Apr2021 18:33:40.670] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.treatEmptyTagsAsAir was corrected from null to its default, false. 
[22Apr2021 18:33:40.670] [Server thread/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key server.fixAdvancementLoading was corrected from null to its default, true. 
[22Apr2021 18:33:40.691] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Preparing level "world"
[22Apr2021 18:33:40.832] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Preparing start region for dimension minecraft:overworld
[22Apr2021 18:33:41.671] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[22Apr2021 18:33:41.671] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[22Apr2021 18:33:41.955] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 0%
[22Apr2021 18:33:44.292] [Netty Server IO #1/INFO] [net.minecraftforge.fml.server.ServerLifecycleHooks/SERVERHOOKS]: Disconnecting Player (server is still starting): Server is still starting! Please wait before reconnecting.
[22Apr2021 18:33:44.510] [Netty Server IO #2/INFO] [net.minecraftforge.fml.server.ServerLifecycleHooks/SERVERHOOKS]: Disconnecting Player (server is still starting): Server is still starting! Please wait before reconnecting.
[22Apr2021 18:33:45.009] [Worker-Main-10/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.009] [Worker-Main-11/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.011] [Worker-Main-11/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.011] [Worker-Main-11/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.012] [Worker-Main-11/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.012] [Worker-Main-9/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 83%
[22Apr2021 18:33:45.340] [Worker-Main-8/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 87%
[22Apr2021 18:33:45.833] [Worker-Main-13/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 87%
[22Apr2021 18:33:46.355] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 87%
[22Apr2021 18:33:46.958] [Worker-Main-9/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 87%
[22Apr2021 18:33:47.108] [Netty Server IO #3/INFO] [net.minecraftforge.fml.server.ServerLifecycleHooks/SERVERHOOKS]: Disconnecting Player (server is still starting): Server is still starting! Please wait before reconnecting.
[22Apr2021 18:33:47.335] [Worker-Main-9/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 89%
[22Apr2021 18:33:47.834] [Worker-Main-13/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Preparing spawn area: 95%
[22Apr2021 18:33:48.172] [Server thread/INFO] [net.minecraft.world.chunk.listener.LoggingChunkStatusListener/]: Time elapsed: 7340 ms
[22Apr2021 18:33:48.173] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Done (7.514s)! For help, type "help"
[22Apr2021 18:33:52.189] [Server thread/INFO] [net.minecraft.network.login.ServerLoginNetHandler/]: com.mojang.authlib.GameProfile@16ce8b72[id=<null>,name=Kraftygames,properties={},legacy=false] (/127.0.0.1:61362) lost connection: Disconnected
[22Apr2021 18:39:48.931] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer/]: Stopping the server
[22Apr2021 18:39:48.980] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Stopping server
[22Apr2021 18:39:48.981] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving players
[22Apr2021 18:39:48.981] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving worlds
[22Apr2021 18:39:48.982] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[world]'/minecraft:overworld
[22Apr2021 18:39:49.153] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (world): All chunks are saved
[22Apr2021 18:39:49.153] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_nether
[22Apr2021 18:39:49.155] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[22Apr2021 18:39:49.155] [Server thread/INFO] [net.minecraft.server.MinecraftServer/]: Saving chunks for level 'ServerLevel[world]'/minecraft:the_end
[22Apr2021 18:39:49.156] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
[22Apr2021 18:39:49.168] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (world): All chunks are saved
[22Apr2021 18:39:49.174] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (DIM-1): All chunks are saved
[22Apr2021 18:39:49.174] [Server thread/INFO] [net.minecraft.world.server.ChunkManager/]: ThreadedAnvilChunkStorage (DIM1): All chunks are saved
